src/setupTests.ts

```diff
@@ -31,13 +37,17 @@ Object.defineProperty(document, 'fonts', {
 window.org = 'ttd';
 window.app = 'test';
 window.instanceId = 'test-instance-id';
-window.logError = jest.fn();
-window.logWarn = jest.fn();
-window.logInfo = jest.fn();
-window.logErrorOnce = jest.fn();
-window.logWarnOnce = jest.fn();
-window.logInfoOnce = jest.fn();
-jest.setTimeout(10000);
+
+window.logError = (...args) => {
+  throw new Error(args.join(' '));
+};
+window.logWarn = window.logError;
+window.logInfo = window.logError;
+window.logErrorOnce = window.logError;
+window.logWarnOnce = window.logError;
+window.logInfoOnce = window.logError;
+
+jest.setTimeout(env.parsed?.JEST_TIMEOUT ? parseInt(env.parsed.JEST_TIMEOUT, 10) : 10000);
```

Comment:
When I added around 550 unit tests for the `formatDateLocale` function, I had to increase this timeout to 15000, as the unit tests kept on timing out on CI. :innocent:

Reply:
😱
I'll change the default here to 15000 then. That's probably what causes the merge conflict as well, I guess.